guides/v2.3/install-gde/system-requirements-tech.md

```diff
@@ -88,6 +88,20 @@ For more information, see [Required PHP settings]({{ page.baseurl }}/install-gde
 
 	*	[More information about `repo.magento.com`]({{ site.baseurl }}/guides/v2.1/release-notes/tech_bull_tls-repo.html)
 
+### Required system dependencies
+
+Magento uses system tools for some of its operations. These applications should be installed:
```

Comment:
Should or must? This is in the "Required system dependencies" section.

Reply:
Good point.